exercises/concept/vehicle-purchase/.docs/hints.md

```diff
@@ -0,0 +1,20 @@
+# Hints
+
+## General
+
+- You can accomplish all of these tasks without using an `if` statement.
+
+## 1. Determine if you will need a drivers license
+
+- Use the [equality operator][cmp] to check whether the first field equals a certain string.
+- Use a a regular expression or the `index` function to check if the second field contains the required substrings.
```

Comment:
Does suggesting `index()` make this more confusing or less confusing? I generally find fewer options tend to be less confusing when exploring something new.